locales/en.default.schema.json

```
@@ -1271,6 +1293,33 @@
           "label": "Desktop image width",
           "info": "Image is automatically optimized for mobile."
         },
+        "image_shape": {
+          "options__1": {
+            "label": "Default"
+          },
+          "options__2": {
+            "label": "Circle"
+          },
+          "options__3": {
+            "label": "Arch"
+          },
+          "options__4": {
+            "label": "Parallellogram"
+          },
+          "options__5": {
+            "label": "Diamond"
+          },
+          "options__6": {
+            "label": "Right Chevron"
+          },
+          "options__7": {
+            "label": "Left Chevron"
+          },
+          "options__8": {
+            "label": "Blob"
+          },
```

```suggestion
        "image_shape": {
          "options__1": {
            "label": "Default"
          },
          "options__3": {
            "label": "Arch"
          },
          "options__8": {
            "label": "Blob"
          },
          "options__7": {
            "label": "Chevron left"
          },
          "options__6": {
            "label": "Chevron right"
          },                 
          "options__2": {
            "label": "Circle"
          },
          "options__5": {
            "label": "Diamond"
          },          
          "options__4": {
            "label": "Parallelogram"
          },
```

I would suggestion to order the options alphabetically like we do for `Icons`. I know we might need to update the numbers to keep the same order from the locales file. We can keep `Default` the first like we do with `None`.